800.renames-and-merges/fonts/a.yaml

```diff
@@ -80,6 +80,7 @@
     - amiri-fonts
     - fonts-hosny-amiri
     - fonts-ttf-amiri
+    - texlive-amiri
```

Comment:
I don't think this is correct. Texlive packages are not the same as plain fonts packages.